### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Jaideep07**
Add a difference between attribute and property in the beginner section

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -326,6 +327,17 @@ There are two methods for it:
 
 <br/>
 
+## 7. Difference between Attribute and Property
+- Attributes provide additional information about the HTML elements, attributes are always specified in the start tag and they usually come in name/value pairs.
+- Properties are the values associated with a JavaScript object, these objects are collection of unordered properties and these properties can be usually be changed, added, and deleted.
```
Comment (annu12340) [inline docs/README.md:379]: Properties are accessed from DOM (Document Object Model) nodes.

Could you add this point also

File: docs/README.md [hunk 2]
```diff
@@ -326,6 +327,17 @@ There are two methods for it:
...
+
+  While writing HTML code, we can define attributes on our HTML elements. Then, once the browser parses our code, a corresponding DOM(Document Object Model) node will be created. This node is an object, and therefore it has properties.
+  
+For example:  
+  ```
+  <input id="inputId" type="text" value="Name:">
+  ```
+The id property is a reflected property for the id attribute.Getting the property reads the attribute value, and setting the property writes the attribute value.
```
Comment (annu12340) [inline docs/README.md:388]: Everything else looks good 👍

**Review by annu12340 — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (Jaideep07) [reply docs/README.md:379]: 👍🏻

**Comment by annu12340** [comment]
Kindly resolve the conflicts

**Comment by Jaideep07** [comment]
done

**Comment by annu12340** [comment]
Last and final change 😅 
Pls resolve the conflicts

**Comment by Jaideep07** [comment]
Done

**Comment by annu12340** [comment]
Thanks for the pr :)

**Comment by Jaideep07** [comment]
Thank you😍

**Pull request merged**